<?xml version="1.0" ?>

<launch>
    <arg name="map_frame" default="map" />
    <arg name="laser_in" default="/RL/scan" />
    <arg name="odom_in" default="/truth_map_posestamped" />
    <arg name="cmd_out" default="/X1/x1_velocity_controller/cmd_vel" />
    <arg name="joy" default="/joy" />
    <arg name="goal_in" default="/move_base_simple/goal" />
    <arg name="linear_scale" default="1.5" />
    <arg name="angular_scale" default="0.8" />

    <node pkg="subt_rl" type="goal_nav_tf_rdpg.py" name="run_rdpg_goal" output="screen">
      <param name="frame" value="$(arg map_frame)" />
      <param name="linear_scale" value="$(arg linear_scale)" />
      <param name="angular_scale" value="$(arg angular_scale)" />

      <remap from="laser_in" to="$(arg laser_in)" />
      <remap from="odom_in" to="$(arg odom_in)" />
      <remap from="cmd_out" to="$(arg cmd_out)" />
      <remap from="joy" to="$(arg joy)" />
      <remap from="goal_in" to="$(arg goal_in)" />
    </node>

</launch>
